.snippets/code/infrastructure/setup-secure-wss/apache2-config.md

```diff
@@ -0,0 +1,7 @@
+```apacheconf
```

Comment:
All snippets should be under the folder `./snippets/code/infrastructure/running-a-node/setup-secure-wss`